src/Tempest/Generation/src/ClassManipulator.php

```diff
@@ -37,7 +45,11 @@ public function save(string $path): self
             mkdir($dir, recursive: true);
         }
 
-        file_put_contents($path, $this->print());
+        $is_success = (bool) file_put_contents($path, $this->print());
```

Comment:
rename to `$isSuccess`

Reply:
Oops, interesting that Rector or CS Fixer not fixed it 🤔

Reply:
Fixed 😄